perllib/FixMyStreet/App/Model/PhotoSet.pm

```diff
@@ -71,6 +73,15 @@ has storage => (
     }
 );
 
+has symlinkable => (
+    is => 'ro',
+    lazy => 1,
+    default => sub {
+        my $cfg = FixMyStreet->config('PHOTO_STORAGE_OPTIONS');
+        return $cfg ? $cfg->{SYMLINK_FULL_SIZE} : 0;
```

Comment:
This should be documented in `docs/customising/config.md`.

Should there be some way for the storage backend to indicate whether it supports symlinks? S3 doesn't, for example, so there might need to be some sanity checking of the config too.